.clang-format

```diff
@@ -1,3 +1,5 @@
+## I'm a little teapot, short and stout.
```

Comment:
Need to make changes in syntax with proper naming convention

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
Good job, this is what i am looking for as a fix.

---
**Sema Reaction:** :ok_hand: This code looks good

Reply:
Good job, i think it will not impact our existing functionality

---
**Sema Reaction:** :ok_hand: This code looks good | **Sema Tags:** Fault-tolerant

Reply:
Good fault-tolerant

---
**Sema Reaction:** :ok_hand: This code looks good | **Sema Tags:** Fault-tolerant

Reply:
Never use arguments, opt to use rest syntax ... instead. eslint: prefer-rest-params

Reply:
fix
Never use arguments, opt to use rest syntax ... instead. eslint: prefer-rest-params

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
Fix this variable - not maintainable 

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Not maintainable

Reply:
Looks good, now maintainable 

---
**Sema Reaction:** :ok_hand: This code looks good | **Sema Tags:** Maintainable

Reply:
Needs a fix. Not maintainable.

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Not maintainable

Reply:
Looks good. Readable.

---
**Sema Reaction:** :ok_hand: This code looks good | **Sema Tags:** Readable

Reply:
nit: Better punctuations.

---
**Sema Reaction:** :ok_hand: This code looks good | **Sema Tags:** Unreadable, Inelegant, Brittle